### ISSUE TITLE
Cant change color settings for pane or slider??

### ISSUE THREAD

**Issue by astroben85** [jamesflorentino/nanoScrollerJS#259]
Copied this into .css and linked in the head section:
.nano { background: black; width: 660px; height: 500px; border-radius: 7px; }
.nano .nano-content { padding: 0px 10px 10px 10px; }
.nano .nano-pane  { background: black; }
.nano .nano-slider { background: #ddd; }
.nano {
  position : relative;
  width    : 100%;
  height   : 100%;
  overflow : hidden;
}
.nano > .nano-content {
  position      : absolute;
  overflow      : scroll;
  overflow-x    : hidden;
  top           : 0;
  right         : 0;
  bottom        : 0;
  left          : 0;
}
.nano > .nano-content:focus {
  outline: thin dotted;
}
.nano > .nano-content::-webkit-scrollbar {
  display: none;
}
.has-scrollbar > .nano-content::-webkit-scrollbar {
  display: block;
}
.nano > .nano-pane {
  background : rgba(0,0,0,.25);
  position   : absolute;
  width      : 10px;
  right      : 0;
  top        : 0;
  bottom     : 0;
  visibility : hidden\9; /\* Target only IE7 and IE8 with this hack _/
  opacity    : .01;
  -webkit-transition    : .2s;
  -moz-transition       : .2s;
  -o-transition         : .2s;
  transition            : .2s;
  -moz-border-radius    : 5px;
  -webkit-border-radius : 5px;
  border-radius         : 5px;
}
.nano > .nano-pane > .nano-slider {
  background: #444;
  background: rgba(0,0,0,.5);
  position              : relative;
  margin                : 0 1px;
  -moz-border-radius    : 3px;
  -webkit-border-radius : 3px;
  border-radius         : 3px;
}
.nano:hover > .nano-pane, .nano-pane.active, .nano-pane.flashed {
  visibility : visible\9; /_ Target only IE7 and IE8 with this hack */
  opacity    : 0.99;
}

Appears to work as I can change the background color and add border-radius in .nano but nothing changes when I alter ".nano .nano-pane  { background: black; } and
.nano .nano-slider { background: #ddd; }" 

Am I missing something obvious? Heres my .index file also without <>:
<!DOCTYPE html>
head
  titleIll Komponent</title>
  link rel="stylesheet" type="text/css" href="css/stylesheet.css"
  link rel="stylesheet" type="text/css" href="css/nanoscroller.css"
  meta name="viewport" content="width=device-width"
  script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script
  link href='http://fonts.googleapis.com/css?family=Open+Sans+Condensed:300' rel='stylesheet' type='text/css'
  link type="text/css" rel="stylesheet" href="css/sideviewmenu.css" /

  script src="js/sideviewmenu.js"
  /***********************************************
- Page Sideview Menu- (c) Dynamic Drive DHTML code library (www.dynamicdrive.com)
- This notice MUST stay intact for legal use
- Visit Dynamic Drive at http://www.dynamicdrive.com/ for this script and 100s more
  ***********************************************/
  /script
  
  script
    jQuery(function(){ // on DOM Load
      sideviewmenu({ // call sideviewmenu() function
        menuid: 'sideviewmenu',
        onopenclose:function(state){ // state is either "open" or "closed"
          // custom code
        }
      })
    })
  /script
  
  script src="js/nanoscroller.js"></script
  
  </head
  
  <body
  
    <header
      <div
        <img id="background" title="" alt="" src="img/background.jpg"/
      </div 
  
  ```
  <div id="page"
    <h1>-illKomponent<div class="toggler" onClick="sideviewmenu.toggle()"></div></h1>
  </div
  ```
  
    </header
  
    <aside
      <div id="about" class="nano"
        <div class="nano-content" 
          <h2>New</h2>
          <iframe src="http://player.vimeo.com/video/98864327?portrait=0" width="590" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe 
          <p>A short film made in 48 hours for the Raindance 48 Hour Film Challenge Portsmouth.</p>
  
  ```
      <p>"Two people come to the end of a long journey."</p
  
      <p>Cast:
      Anna - Janina Blohm-Sievers
      Michael - Ben Caswell
      Nick - Nick Downes</p>
  
      <p>Written, Produced and Directed by: Riyadh Haque & John McGinty</p>
  
      <p>DoP//Co-Editor: Riyadh Haque
      Co-Editor//Colourist//Gaffer//AC - Daniel Hawkins
      Sound & Original Music: Isaac Ray
      Producer & Clapper: Bensalem Mitchell</p>
  
      <p>Obstructions and key information:
      Line: "I've never looked at it like that before";
      A location that is notable in Portsmouth;
      Use a piece of stock from Pond5</p>
  
      <iframe src="http://player.vimeo.com/video/91904317?portrait=0" width="590" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
  
      <p>Details to include:</p>
  
      <p>Title: 33 REVS PER MINUTE
      Line: "Anyway, it seemed like a good time to launch the Hillis Cycle."
      Prop: KEYBOARD_ WE SEE A CHARACTER REPLACE A MISSING KEY ON A KEYBOARD</p>
    </div
  </div
  ```
  
    </aside
  
  </body

</html

Apologies if this is too much info or not enough, I really am just starting off new with my coding adventures.
